### PR TITLE
Add jonesdoc-raleigh pin to the map.

### PR DESCRIPTION
**Pull request by jonesdoc**
Puting Fuquay on the map!

### CODE REVIEW

File: _pins/jonesdoc.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+...
+githubhandle:jonesdoc
+latitude:35.584324
+longitude:-78.800013
+...
```
Comment (a-a-ron) [inline _pins/jonesdoc.json]: Hey @jonesdoc, make sure your format follows the below example exactly. 

```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```
Replace the `...` with `---` and add a space after the `:`. After those changes your build should pass.

**Comment by hectorsector** [comment]
@califocation appreciate the help ❤️. However, please let each individual fix their own branch so they can get the ✨ feeling of learning and overcoming obstacles. You can contribute by commenting on any open PRs.

File: _pins/jonesdoc.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+githubhandle: jonesdoc
+latitude: 35.584324
+longitude: -78.800013
+...
```
Comment (hectorsector) [inline _pins/jonesdoc.json]: @jonesdoc make sure you add `---` to the top and bottom of the file (not `...`), and include a capital `H` in `githubHandle`

**Comment by hollenberry** [comment]
Hey @jonesdoc  ! Looks like you've gotten good advice from Hector above! If you take action on those changes, you'll be set. 👍

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.

**Comment by crichID** [comment]
@jonesdoc I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/5076/commits/f4b7132c157d4583fdfe1f51c4e7e62e2d4192dd. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 21st, I will go ahead and do it just to keep the repository clean!

**Pull request merged**